src/components/editor/ResourceTemplate.jsx

```diff
@@ -39,17 +39,21 @@ class PropertiesWrapper extends Component{
       border: '1px dashed',
       padding: '10px',
     }
-    return (
-      <div className='PropertiesWrapper' style={dashedBorder}>
-        <p>BEGIN PropertiesWrapper</p>
-          <div>
-            {this.props.propertyTemplates[0].map(function(pt, index){
-              return <PropertyTemplate key={index} propertyTemplates={[pt]}/>
-            })}
-          </div>
-        <p>END PropertiesWrapper</p>
-      </div>
-    )
+    if (this.props.propertyTemplates.length == 0 || this.props.propertyTemplates[0] == {}) {
```

Comment:
Maybe we need a global `isEmpty()` function. This is something I used in the ValueConstraints component and it may have broader use. See https://coderwall.com/p/_g3x9q/how-to-check-if-javascript-object-is-empty

Reply:
Interesting idea.  I wonder what @jermnelson thinks.

Reply:
@jgreben and I discussed F2F; plan is to use #158 ultimately